File: Plugin/Example 2.vb
Public Class Example_2

    Implements Versidyne.Plugins.Plugin

    Private ObjHost As Versidyne.Plugins.Host

    Public Sub Initialize(ByVal Host As Versidyne.Plugins.Host) Implements Versidyne.Plugins.Plugin.Initialize

        ObjHost = Host

    End Sub

    Public ReadOnly Property Name() As String Implements Versidyne.Plugins.Plugin.Name

        Get

            Return "Example Plugin 2"

        End Get

    End Property

    Public Function Main(ByVal Args As Array) As Integer Implements Versidyne.Plugins.Plugin.Main

        ObjHost.ShowFeedback("Main function test.")

        Return 0

    End Function

End Class

File: Plugin/Example 2.cs
// VBConversions Note: VB project level imports
using System.Collections.Generic;
using System;
using Versidyne.Plugins;
using System.Diagnostics;
using System.Data;
using System.Xml.Linq;
using Microsoft.VisualBasic;
using System.Collections;
using Versidyne;
using System.Linq;
// End of VB project level imports


namespace Plugin
{
	public class Example_2 : Versidyne.Plugins.Plugin
	{
		
		
		private Versidyne.Plugins.Host ObjHost;
		
		public void Initialize(Versidyne.Plugins.Host Host)
		{
			
			ObjHost = Host;
			
		}
		
		public string Name
		{
			
			get
			{
				
				return "Example Plugin 2";
				
			}
			
		}
		
		public int Main(Array Args)
		{
			
			ObjHost.ShowFeedback("Main function test.");
			
			return 0;
			
		}
		
	}
	
}
